_sass/custom/custom.scss

```diff
@@ -73,3 +74,44 @@ p, ul, ol {
     margin: 1.7rem 0 1rem 0;
     font-size: 1.6rem;
 }
+
+.custom-card-wrapper {
```

Comment:
- Move card styles to the _card.scss file (to consolidate all card component styles).
- Remove .custom-card-wrapper class from html, use '.blog .card' class instead for custom blog card styling.

Reply:
Ensure with the style changes that the inconsistent 'rounded corners' of the blog card have the same border-radius.

Reply:
Yeah, this should probably go in the card stylesheet, but I gave it a different class name instead of specificity because we discussed that the "component" could be "reusable" and copied for other pages like book clubs, etc

Reply:
> Ensure with the style changes that the inconsistent 'rounded corners' of the blog card have the same border-radius.

Not sure what you mean by this

Reply:
Yeah, the card styles should probably also be refactored once moved into the .card file.
I.e. all shared card style elements should go under the .card style block.
If we have elements specific to a component such as the bottom byline in blog card, we can use '.blog .card' class for that for now.

But if this PR needs to go out soon, we can create a separate to refactor the card styles, since those changes affect other pages.

Reply:
<img width="771" alt="Screenshot 2024-04-28 at 08 54 41" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/26842896/bfb23967-dc63-45e8-a0bd-30ac122cd925">

I mean that the top rounded corners differs from the bottom ones :)

Reply:
For the blog card, reducing the specificity to .blog .card caused some of the more specific styling to be overriden by styles with higher specificity elsewhere. Would leave this as a specific class for now (renamed to ".blog-card")